app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -276,21 +316,22 @@ class PatientSummaryScreenController @Inject constructor(
         .map { (_, _, uuid) -> { ui: Ui -> ui.showScheduleAppointmentSheet(patientUuid = uuid) } }
 
     val backClicksWithBpNotSaved = backClicks
-        .withLatestFrom(mergedBpSaves, callers)
-        .filter { (_, saved, _) -> saved.not() }
-        .map { (_, _, caller) ->
+        .withLatestFrom(mergedBpSaves, callers, patientSummaryResultItem) { _, saves , uuid , result -> Triple(saves, uuid, result) }
```

Comment:
Could you please rename `uuid` -> `caller` since it's an instance of `PatientSummaryCaller` and not a `UUID`?